org.mixedrealitytoolkit.core/Editor/Editors/StatefulInteractableEditor.cs

```diff
@@ -91,12 +95,9 @@ protected void DrawProperties(bool showToggleMode)
 
             StatefulInteractable interactable = target as StatefulInteractable;
 
-            bool interactableActive = EditorGUILayout.Toggle(new GUIContent("Is Interactable", "Convenience alias for StatefulInteractable.enabled"), interactable.enabled);
-
-            if (interactableActive != (target as StatefulInteractable).enabled)
+            if (IsInteractable != null)
```

Comment:
Curiosity: is it really necessary? In what case IsInteractable can be null?

Reply:
@anonymous2585 
You're probably right, it isn't necessary and would never be null.  However, I couldn't find any official documentation from Unity on obtaining the serialized property for .enabled, even though using "m_Enabled" is how the Editor class finds it itself.  But not being documented, that I could find, made me consider that it would be safer to add the check just in case Unity decides to change things.  The check could probably be removed, I'm cool either way.